packages/mock-block-dock/src/use-mock-block-props/use-mock-datastore.ts

```diff
@@ -66,6 +67,17 @@ export const useMockDatastore = (
   const createEntity: EmbedderGraphMessageCallbacks["createEntity"] =
     useCallback(
       async ({ data }) => {
+        if (readonly) {
+          return {
+            errors: [
+              {
+                code: "FORBIDDEN",
+                message: "Operation can't be carried out in readonly mode",
+              },
+            ],
+          };
+        }
```

Comment:
You can save a bit of space by declaring the readonly error at the top of the file and then referencing it, since it is the same in every function, e.g.
```typescript
const readonlyErrorReturn: {
  errors: [{ code: "FORBIDDEN"; message: string }];
} = {
  errors: [
    {
      code: "FORBIDDEN",
      message: "Operation can't be carried out in readonly mode",
    },
  ],
};

...

if (readonly) {
  return readonlyErrorReturn;
}

Reply:
Thanks! Added in [f020a5c](https://github.com/blockprotocol/blockprotocol/pull/484/commits/f020a5c352818f6861429fe3955c3ab80ee3433a)